nasa/earthdata-search
# EDSC-4251: Fixed progress bar for SWODLR jobs

# Overview

### What is the feature?

Fixed it so that complete SWODLR jobs now show the progress bar at 100%.

### What is the Solution?

Added logic to account for the order being a SWODLR order.

### What areas of the application does this impact?

Progress bar in the downloads page.

# Testing

### Reproduction steps

- **Environment for testing:** sit
- **Collection to test with:** C2799438271-POCLOUD

1. create a SWODLR project and add 2 granules 
2. click "Download"
3. on the downloads page wait for the job to complete
4. observe the blue progress bar go to 100% when it does complete.

### Attachments

![image](https://github.com/user-attachments/assets/1817d79b-ddad-415a-992f-2585aaa9e14a)
![image](https://github.com/user-attachments/assets/75904f7f-31ac-4ac8-ba6b-c4b3cfabd07b)
![image](https://github.com/user-attachments/assets/44bbe581-eb74-4827-a3a1-6133075f163e)


# Checklist
